subprojects/docs/src/docs/dsl/dsl.xml

```diff
@@ -159,9 +159,6 @@
             <tr>
                 <td>org.gradle.api.file.SourceDirectorySet</td>
             </tr>
-            <tr>
-                <td>org.gradle.api.tasks.incremental.IncrementalTaskInputs</td>
-            </tr>
```

Comment:
Should we keep `IncrementalTaskInputs` in the DSL manual? It is used even in Gradle tasks.

Reply:
I re-added it to the DSL doc.